<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>291</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>335</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>182</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit_1">
      <property name="plainText">
       <string>Количество запрошенных замен текста</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="conwPT"/>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="coctPT"/>
    </item>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit_0">
      <property name="plainText">
       <string>Количество новых слов</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit_2">
      <property name="plainText">
       <string>Количество запрошенных переводов с английского на русский</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="coawPT"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
